src/Sulu/Bundle/MediaBundle/Media/Storage/S3Storage.php

```diff
@@ -32,9 +32,11 @@ class S3Storage extends FlysystemStorage
      */
     private $bucketName;
 
-    public function __construct(FilesystemInterface $filesystem, int $segments)
+    private $url;
+
+    public function __construct(FilesystemInterface $filesystem, int $segments, $url)
     {
-        parent::__construct($filesystem, $segments);
+        parent::__construct($filesystem, $segments, $url);
```

Comment:
Parent call here not required:

```suggestion
        parent::__construct($filesystem, $segments);
```